PWGHF/TableProducer/HFCandidateCreatorX.cxx

```diff
@@ -233,6 +262,8 @@ struct HFCandidateCreatorXMC {
                aod::McParticles const& particlesMC)
   {
     int indexRec = -1;
+    int xPdgCode = 9920443;
+    int jpsiPdgCode = 443;
```

Comment:
You are supposed to pick these from `pdg::Code`.

Reply:
Done

Reply:
Done